data/appdata-xml.m4

```diff
@@ -36,7 +36,7 @@ AU_DEFUN([APPDATA_XML],
          AS_IF([test "x$enable_appdata_validate" = "xyes"],
                [AC_MSG_ERROR([AppData validation was requested but appstream-util was not found])])])
 
-  AC_SUBST([appdataxmldir], [${datadir}/appdata])
+  AC_SUBST([appdataxmldir], [${datadir}/metainfo])
```

Comment:
Do you have a list of packages that ship files in metainfo handy? I can do the base-filesystem changes myself but i need to add some reasoning to the commit.